doc/api/debugger.md

```diff
@@ -28,7 +28,7 @@ enable a breakpoint at that position in the code:
 
 ```js
 // myscript.js
-x = 5;
+global.x = 5;
```

Comment:
`var x = 5` and the same in the output examples would be more representative of usage IMO.